e2e/template.go

```diff
@@ -452,7 +452,7 @@ func baseTemplate(location string) *datamodel.NodeBootstrappingConfiguration {
 func getHTTPServerTemplate(podName, nodeName string, isAirgap bool) string {
 	image := "mcr.microsoft.com/cbl-mariner/busybox:2.0"
 	if isAirgap {
-		image = "aksvhdtestcr.azurecr.io/aks/cbl-mariner/busybox:2.0"
+		image = fmt.Sprintf("%s.azurecr.io/aks/cbl-mariner/busybox:2.0", PrivateACRName)
```

Comment:
is there anything that going push the image into ACR?

If the `PrivateACRName` is going to change or the ACR is deleted then tests will break, right?

Should the image be pulled from mcr and pushed to acr if it doesn't exist?

Reply:
upstream pipelines deal with taking newly published images and pushing them to MCR. Because ACR has a cache rule to MCR, as long as it exists in MCR we will be able to grab it. 

If the ACR is deleted / a name change then yes the tests will break, but the ACR should get re-created during e2e setup.

Reply:
>Because ACR has a cache rule to MCR, as long as it exists in MCR we will be able to grab it.

Is there something pushed the image into ACR?

Reply:
Yeah there is the aks upstream team who has pipelines that get triggered when the open source upstream github repos get new versions. When that happens the images get pushed to MCR, during thoes triggered pipelines, after the aks upstream team does some extra validation. But its supposed to happen basically the same time as the open source github publishes a new version.